### DISCUSSION TITLE
Why is v3.x downloaded 25 times less than v2.x ?

### DISCUSSION THREAD

**Discussion by cstavaru** [node-fetch/node-fetch#1709] [Q&A]
It seems strange to me. You would think the developers of this package have noticed this issue until now, and tried to somehow fix it.

**Comment by LinusU** [comment]
I don't find it too strange considering that 1) many people are still stuck on CommonJS, especially older code bases, and 2) many people that have support for ESM are also running a version of Node.js new enough to support `fetch` without a package at all.

> [...] noticed this issue until now, and tried to somehow fix it.

What issue? 🤔
